requirements.txt

```diff
@@ -1,3 +1,6 @@
 requests>=2.5.0,<2.6
 websocket-client
+beautifulsoup4
+lxml
```

Comment:
Can you please pin this to some major version, like `requests` so we don't get any sudden surprises when upstream lib indtroduces any breaking changes.